src/components/GradientPicker/index.js

```diff
@@ -82,6 +83,10 @@ const GradientPicker = ({
 		setActiveColorId(id);
 	};
 
+	useEffect(() => {
+		onColorStopSelect(activeColorId);
```

Comment:
Instead of triggering the given callback with activeColorId, Lets send the active color as whole, having the consume what he needs from it ( He could access `color.id`)

Reply:
This is a good idea.

Reply:
Lets make it happen than :) so we could merge this one.